### DISCUSSION TITLE
Why having KeepAliveTime?

### DISCUSSION THREAD

**Discussion by sts-odoo** [coder/code-server#2934] [Q&A] [answered]
I was wondering what is the reasoning behind having the KeepAliveCheck every 5 seconds ?

Would you accept a PR setting the KeepAliveTime and other protocol constants within the config file?

**Comment by code-asher** [comment, marked as answer]
I think it would definitely be best to ask this upstream in VS Code! I think it's probably something we wouldn't want to patch downstream.
